colorBorderIIFE.js

```diff
@@ -1,73 +1,70 @@
-    }
-    });
-  };
+    newCarLot.resetBGAndBorder = function() {
+        container.addEventListener("click", function(event) {
+            var carDiv = document.getElementsByClassName('car');
+            for (var i = 0; i < carDiv.length; i++) {
+                carDiv[i].classList.remove('selectedCar');
+                carDiv[i].style.borderWidth = "medium";
+                carDiv[i].style.backgroundColor = "#f0f0f0"
+            };
+        });
+    };
 
-  /* Adds event listener to container div that gives the selected car element a class of "selectedCar". This function
-  exists to make a later function actually work. */
+    /* Adds event listener to container div that gives the selected car element a class of "selectedCar". This function
+    exists to make a later function actually work. */
 
-  newCarLot.changeClassForStyle = function() {
-    container.addEventListener("click", function(event) {
-      var carDiv = document.getElementsByClassName('car');
-        for (var i = 0; i < carDiv.length; i++) {
-        event.target.parentNode.classList.add('selectedCar');
-        }
-      }
-    )
-  };
+    newCarLot.changeClassForStyle = function() {
+        container.addEventListener("click", function(event) {
+            var carDiv = document.getElementsByClassName('car');
+            for (var i = 0; i < carDiv.length; i++) {
+                event.target.parentNode.classList.add('selectedCar');
+            };
+        });
+    };
 
-  /* Adds event listener to container element that loops through all car elements on page and checks for the element
-  passed into the function. If one is found, it modifies the border to a preset value and changes the background 
-  color to one also passed in as the second arguement of the function. */
+    /* Adds event listener to container element that loops through all car elements on page and checks for the element
+    passed into the function. If one is found, it modifies the border to a preset value and changes the background 
+    color to one also passed in as the second arguement of the function. */
 
-  newCarLot.styleChanger = function(element, color) {
-    container.addEventListener("click", function(event) {
-      var carDiv = document.getElementsByClassName('car');
-        for (var i = 0; i < carDiv.length; i++) {
-          if (carDiv[i].classList.contains(element)) {
-            carDiv[i].style.borderWidth = "thick";
-            carDiv[i].style.backgroundColor = color;
-          }
-        }
-    })
-  };
+    newCarLot.styleChanger = function(element, color) {
+        container.addEventListener("click", function(event) {
+            var carDiv = document.getElementsByClassName('car');
+            for (var i = 0; i < carDiv.length; i++) {
+                if (carDiv[i].classList.contains(element)) {
+                    carDiv[i].style.borderWidth = "thick";
+                    carDiv[i].style.backgroundColor = color;
```

Comment:
I would prefer to see all the style changes happen by adding a class that has these styles attached to it in the CSS.